libbeat/docs/outputconfig.asciidoc

```diff
@@ -198,7 +198,7 @@ operation doesn't succeed after
 ===== bulk_max_size
 
 The maximum number of events to bulk in a single Elasticsearch bulk API index request.
-The default is 50.
+The default is 200.
```

Comment:
So we've got multiple output plugins:
- Elasticsearch
- logstash
- console
- file

The default bulk_max_size is used for all output plugins but elasticsearch, which sets the default to 50.

This options sets maximum number of events that can be combined internally into batches and will be publishable by the output plugins =>
- if beat tries to send single events, the events are collected into batches
- if beat tries to publish large batch of event (bigger bulk_max_size), the batch will be split.

 Bigger batch sizes can improve performance due to ammortizing per event sending overhead. On the other hand to big batch sizes can increase processing time such that queues in logstash/elasticsearch can not be processed ->  APIs return errors, connections get killed or publish requests time out. This increases latency and lowers throughput for indexing events.